src/test/acceptance/features/cookies.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Cookies details page
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  I want to view the cookies used by the site
+
+  Scenario: The cookie back link is not shown if we navigate directly to it
+    When I go directly to the cookies page
```

Comment:
I'm not sure the word `directly` adds anything here.

Reply:
I would in this case as how you got to the page is the key part of the feature.